src/functions.php

```diff
@@ -597,11 +597,18 @@ function warning(string $message): void
     }
 }
 
+/**
+ * Writes a message to the output.
+ */
+function write(string $message, int $options = 0): void
+{
+    output()->write(parse($message), false, $options);
+}
+
 /**
  * Writes a message to the output and adds a newline at the end.
- * @param string|array $message
```

Comment:
I don't know that `parse()` "doesn't support" `iterable`. If you pass `iterable`, `parse()` just ignores it because `parse()` only deals with strings, but other types pass through unchanged. Moreover, this was previously typed as `array`. I don't know if it's OK for iterables (including arrays) to pass through `parse()` unchanged because I don't understand what `parse()` does, but if it just fiddles around with newlines, perhaps this doesn't apply to iterables anyway, since the underlying Symfony library will handle this.